3634/1/gems.rb

```diff
@@ -0,0 +1,27 @@
+class Gems
+  attr_reader :names
+
+  def initialize(gems)
+    @names = gems(gems)
+    @paths = []
+  end
+
+  def links
+    paths(@names)
+    @paths
+  end
+
+  private
+
+  def gems(gems)
+    Psych.load_file(gems)['gems']
```

Comment:
What's the `Psych`?

Reply:
This is not added to gemfile

Reply:
what if you duplication gem name in the file?

Reply:
This is a Ruby Std-lib 2.5.1 module